opentelekomcloud/services/cbr/resource_opentelekomcloud_cbr_vault_v3.go

```diff
@@ -371,6 +365,10 @@ func resourceCBRVaultV3Create(ctx context.Context, d *schema.ResourceData, meta
 		}
 	}
 
+	if err := common.UpdateResourceTags(client, d, "vault", d.Id()); err != nil {
```

Comment:
Please manage tags along with vault creation, there's no need to send an extra request